### DISCUSSION TITLE
Forcing 'Page scrolling' mode by default

### DISCUSSION THREAD

**Discussion by SirClickALot** [mozilla/pdf.js#17769] [Q&A]
I am using a Drupal module that appears to do little more than 'throw' a PDF file at the pdf.js viewer.

I say 'little more' but actually it's doing a perfect job.

Almost...

I just need the viewer to default to 'Page scrolling' mode by default.
Since there does not seem to be any way to effect this using the module, I was wondering if I can create a custom (shaped up) version of this library that does default to that mode.

Thanks

**Comment by catmans1** [comment]
@SirClickALot 
If you're using `viewer.js` with pdfjs version `3.11.174`
You can hide another `Scroll Mode` in `viewer.html`
And set Page scroll mode in `PDFViewerApplication` (Page scroll mode is 3)  in your js after the pdf viewer rendered
